themebuilder/modules/modules-handler.js

```diff
@@ -0,0 +1,96 @@
+class ModulesHandler {
+    constructor(widgetsList) {
+        this.SPECIAL_COMMENT = "tb_widgets_list";
+        this.FILE_FOR_PARSING = "theme.less";
+
+        this.widgets = (widgetsList || []).map(w => w.toLowerCase());
+
+        this.bundledWidgets = [];
+    }
+
+    getWidgetFromImport(importString) {
+        const lastSlashIndex = importString.lastIndexOf("/");
+        const fileName = importString.substr(lastSlashIndex + 1);
+        const dotIndex = fileName.indexOf(".");
+        return fileName.substr(0, dotIndex);
+    }
+
+    availableWidgets(less) {
+        const widgets = [];
+        const widgetsListIndex = less.indexOf(this.SPECIAL_COMMENT);
+
+        if(widgetsListIndex < 0) {
```

Comment:
Let's invert: if widgetsListIndex >= 0 then process `less`. There will be only one return statement.

Reply:
I'll change. But I do not like this because this condition makes the nesting bigger.